packages/frontend/web-ui/src/scss/auth/login/_page.scss

```diff
@@ -0,0 +1,40 @@
+@use '../../base/variables' as var;
+
+.login-form-grid {
+  background-color: var(--cg-text-primary);
+  border-radius: 10px;
+  text-align: center;
+  color: var(--cg-bkg-primary-light);
+  padding: 3rem;
+  margin: 0rem 2rem;
+}
+
+@media (min-width: var.$sm) {
+  .login-form-grid {
+    margin: 0;
+    max-width: 450px;
+  }
+}
+
+.login-title-text-position {
+  padding-bottom: 1.5rem;
+}
+
+.form-login-error {
+  margin-top: 1.5rem;
+  text-align: left;
+}
+
+.login-form-grid .form-fieldset-border-color {
```

Comment:
I think `form-fieldset-border-color` should be `form-text-field` instead. I think you're applying it to all the text fields to apply styles you want to apply to all the form thex fields, am I right?

Reply:
Done! I've renamed the CSS class name.